### PR TITLE
chore: remove unused files

### PR DESCRIPTION
**Pull request by zirain**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: charts/eg/crds/generated/config.gateway.envoyproxy.io_envoyproxies.yaml [hunk 1]
```diff
@@ -1,239 +0,0 @@
----
```
Comment (arkodg) [inline charts/eg/crds/generated/config.gateway.envoyproxy.io_envoyproxies.yaml:1]: can you please remind me why this  isn't needed anymore

Comment (zirain) [reply charts/eg/crds/generated/config.gateway.envoyproxy.io_envoyproxies.yaml:1]: I think it's moved to charts/gateway-helm and will not be update by `make generate manifests`

Comment (arkodg) [reply charts/eg/crds/generated/config.gateway.envoyproxy.io_envoyproxies.yaml:1]: good catch, so `charts/eg` is not longer needed ? 
cc @Xunzhuo

**Review by Xunzhuo — APPROVED** [review]
good catch!

**Pull request merged**